src/ast.rs

```diff
@@ -328,6 +328,27 @@ pub struct CallArg<A> {
     pub value: A,
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct RecordUpdateSpread {
+    pub name: String,
+    pub location: SrcSpan,
+}
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct UntypedRecordUpdateArg<A> {
```

Comment:
I can remove these type parameters now, and just inline `UntypedExpr` and `TypedExpr`